Apps/Playground/macOS/ViewController.mm

```diff
@@ -51,7 +53,12 @@ - (void)viewDidLoad {
 }
 
 - (void)uninitialize {
-    inputBuffer.reset();
+    if (graphics)
+    {
+        graphics->FinishRenderingCurrentFrame();
+    }
+
+    nativeInput = nullptr;
```

Comment:
It might be worth adding a comment on why this isn't a leak. I'm pretty sure it isn't as the JS context owns that memory, but it might be worth calling that out explicitly as seeing a raw pointer just set to null like this is a little jarring.

Reply:
I added a comment